index.html

```diff
@@ -31,13 +32,213 @@ <h2 id="job-title">I am a Software Developer</h2>
+                    <div class="floating">
+                        <div>
+                            <h3 class="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+                        </div>
+                        <ul class="project-1-buttons-ul">
+                            <a href="#"><button class="project-1-buttons">html</button></a>
+                            <a href="#"><button class="project-1-buttons" >bootstrap</button></a>
+                            <a href="#"><button class="project-1-buttons">Ruby</button></a>
+                        </ul>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button class="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+                <div class="project-1 last-project">
+                    <img src="images/Img Placeholder-2.png" alt="">
+                    <div class="floating">
+                        <div>
+                            <h3 class="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+                        </div>
+                        <ul class="project-1-buttons-ul">
+                            <li class="project-1-buttons-li"><a href="#"><button class="project-1-buttons">html</button></a></li>
+                            <li class="project-1-buttons-li"><a href="#"><button class="project-1-buttons">bootstrap</button></a></li>
+                            <li class="project-1-buttons-li"><a href="#"><button class="project-1-buttons">Ruby</button></a></li>
+                        </ul>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button class="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+            </section>
+            <section class="about-me">
+                <div id="about-div">
+                    <h2 id="about-me-title">About Me</h2>
+                </div>
+                <div id="about-me-description-div">
+                    <p id="about-me-description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <a href="#"><button id="get-resume-button">Get My Resume</button></a>
```

Comment:
- [ ] One of the objectives of this project is to understand how to parse Figma designs. We should try our best to make our design match the Figma template. :smiley_cat: 

|Figma design|Your design|
|----------------|-------------------|
|   ![Screenshot from 2022-12-05 10-20-54](https://user-images.githubusercontent.com/65129228/205586153-49543287-42f7-4a87-8b03-35115de724c0.png) | ![Screenshot from 2022-12-05 11-11-50](https://user-images.githubusercontent.com/65129228/205586539-5abb5cad-ef64-4fa4-b127-cf77072e661e.png) |  